packages/pages/src/scaffold/modules/templates.ts

```diff
@@ -5,8 +5,9 @@ export const moduleCode = (
   useTailwind: boolean
 ): string => {
   const tailwind = useTailwind ? ` className="tailwind"` : ``;
-  const formattedModuleName =
-    moduleName.charAt(0).toUpperCase() + moduleName.slice(1);
+  const formattedModuleName = moduleName
```

Comment:
Move the formatting into a function and add tests for it.

Reply:
Oops done